openy_media/modules/openy_media_local_video/config/install/field.field.media.video_local.field_media_local_video.yml

```
@@ -17,7 +17,7 @@ translatable: false
 default_value: {  }
 default_value_callback: ''
 settings:
-  file_extensions: ''
+  file_extensions: 'mp4 webm ogv'
```

```suggestion
  file_extensions: 'mp4 webm ogv avi'
```
you forgot your name @froboy  )))))